perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ModulesUtilsBlImpl.java

```diff
@@ -1043,14 +1043,14 @@ public User getUserFromMessage(PerunSessionImpl sess, String message) throws Int
 			try {
 				user = getPerunBl().getUsersManagerBl().getUserById(sess, userExtSource.getUserId());
 			} catch (UserNotExistsException ex) {
-				log.debug("User from UserExtSource {} can't be found by id in Perun. Probably not exists any more or UserExtSource was audited without proper UserId!", userExtSource);
+				log.info("User from UserExtSource {} can't be found by id in Perun. Probably not exists any more or UserExtSource was audited without proper UserId!", userExtSource);
 				return null;
 			}
 		} else if (user == null && member != null) {
 			try {
 				user = getPerunBl().getUsersManagerBl().getUserById(sess, member.getUserId());
 			} catch (UserNotExistsException ex) {
-				log.debug("User from Member {} can't be found by id in Perun. Probably not exists any more or Member was audited without proper UserId!", userExtSource);
+				log.info("User from Member {} can't be found by id in Perun. Probably not exists any more or Member was audited without proper UserId!", userExtSource);
```

Comment:
I don't see any reason, why this should be the info level.

Reply:
Same like in the situation above.